### PR TITLE
pretty logs ✨

### PR DESCRIPTION
**Pull request by balajmarius**

### CODE REVIEW

**Comment by balajmarius** [comment]
[(2) Format logs on logality](https://trello.com/c/sHQKFcFL/26-2-format-logs-on-logality)

File: app/logality.js [hunk 1]
```diff
@@ -10,8 +10,16 @@ const os = require('os');
 
 const stackTrace = require('stack-trace');
 
+const chalk = require('chalk');
+
```
Comment (thanpolas) [inline app/logality.js]: No need for newlines between 3rd party libs, remove them.

general format:

```
[system requirements]

[3rd party deps]

[local deps]
```

File: app/logality.js [hunk 2]
```diff
@@ -148,17 +193,73 @@ Logality.prototype._getDt = function () {
...
+Logality.prototype._writePretty = function (logContext) {
+  // current level icon and color
+  const config = LEVELS_CONFIG[logContext.level];
+
+  const date = chalk.white(`[${logContext.dt}]`);
+  const message = config.color(`${config.icon} ${logContext.level} - ${logContext.message}`);
+
+  // format logs
+  const logs = this._getLogs(logContext);
+  const output = `\n${date} ${message}\n\n${logs}\n`;
```
Comment (thanpolas) [inline app/logality.js]: do we need the first newline?

File: test/spec/__snapshots__/pretty.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Pretty Logging Will pretty log an object in context 1`] = `
+"
+[37m[2018-05-18T16:25:57.815Z][39m [34mℹ info - hello world[39m
+
```
Comment (thanpolas) [inline test/spec/__snapshots__/pretty.test.js.snap]: that newline is not needed, have the object directly under the log line.

Comment (balajmarius) [reply test/spec/__snapshots__/pretty.test.js.snap]: I added that line to prevent this - http://prntscr.com/jr3he1

**Comment by balajmarius** [comment]
Updated.

**Comment by thanpolas** [comment]
Good Job!

**Pull request merged**